packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -69,7 +73,7 @@ const BuySellHeader = ({ is_visible, table_type, setTableType }) => {
             </div>
             <div className='buy-sell__header-match-ads'>
                 <Checkbox
-                    label={localize('Show the matching ads')}
+                    label={preparedLocalize('Show the matching ads')}
```

Comment:
What is the purpose to add this new function? Besides that, I think this function wouldn't work because of the nature of our translations. Crowdin simply goes through the code and looks for strings inside `localize` function to translate them. When you provide variable inside `localize` it will not be translated as the translation does not happen during runtime.

Reply:
Changed it by putting the label in a seperate Localize container.